### ISSUE TITLE
[BUG] Tooltips from Items posted in Chat close themselves

### ISSUE THREAD

**Issue by Arkios64** [arkayenro/arkinventory#1696]
The popout Tooltip from clicking on any items mentioned in chat (including any loot drops) is automatically closing itself after around a second. 
Disabling the `Item Count` function fixes the issue.

The bug occurs independently from the `Item Count` being displayed, even when first displaying the tooltip and the count isn't loaded yet it closes itself.
If you enable the function with a tooltip currently active it isn't getting updated and doesn't automatically close, only fresh instances after enabling it cause it.

**Comment by arkayenro** [comment]
i did notice that, i'll get around to looking into it once i get tooltip handling in general rebuilt.

**Comment by Arkios64** [comment]
Correction: the first instance, before the function is fully spun up to know the amounts you have, it stays open.
So it's likely something to do with updating the information, if that helps.

**Comment by arkayenro** [comment]
its working fine in classic and wrath so i presume its another of those things that blizzard changed for some reason.  just have to figure it out again.

**Comment by Arkios64** [comment]
I remember this having been an issue multiple times before, just randomly in the middle of a patch cycle; although I don't actually remember how I fixed it the last few times and the usual ways of searching didn't come up with anything either.

**Comment by arkayenro** [comment]
its part of the underlying tooltip issue so would have been in beta, prt, and prepatch.

try 3.10.09 alpha 2

**Comment by Arkios64** [comment]
No, as in, something like this happened occasionally, years back, with completely different causes. Either way, doesn't matter.

Can confirm it works now.

**Comment by Arkios64** [comment]
Wait, I noticed something new: clicking on an achievement in chat has the same issue, except instead of closing the Tooltip it re-opens the last item Tooltip. If no item has been clicked on before the achievement tooltip displays fine.
It only happens with `3.10.09 alpha 2`, so the fix you applied seems to be the cause.
Disabling the `item count` function once again fixes the problem and it only reappears after you click on a new item with `item count` enabled again.

Also, scrolling in `Edit Mode` #1690 isn't working again in `3.10.09 alpha 2`, just in case that isn't intended.

**Comment by arkayenro** [comment]
i knew the way i fixed it would come back to bite me.

and no i didnt intend to break the scrolling, but then i also didnt intend to fix it last time.

**Comment by arkayenro** [comment]
ok, try 3.10.09 alpha 3

theres still one weird issue if you open an item one first, it wont refresh, but every one after that seems fine.

**Comment by Arkios64** [comment]
Oh, I get chat readouts for items being moved around (to different bag spaces and to the character), I'm guessing that's just accicentally left in from testing.

**Comment by arkayenro** [comment]
yeah, i turn them on to check something and sometimes miss turning them off before uploading
